pkg/daemon/daemon.go

```diff
@@ -253,9 +264,28 @@ func (dn *Daemon) addProfileConfig(socketPath string, configFile string, nodePro
 	return nil
 }
 
+// Add fifo scheduling if specified in nodeProfile
+func addScheduling(nodeProfile *ptpv1.PtpProfile, cmdLine string) string {
+	if nodeProfile.PtpSchedulingPolicy != nil && *nodeProfile.PtpSchedulingPolicy == "SCHED_FIFO" {
```

Comment:
why we add the PtpSchedulingPolicy if the only policy here is `SCHED_FIFO` ?

Reply:
If SCHED_OTHER, then no need to change it as that's the default.

Reply:
there are move types

```
 -b, --batch          set policy to SCHED_BATCH
 -d, --deadline       set policy to SCHED_DEADLINE
 -f, --fifo           set policy to SCHED_FIFO
 -i, --idle           set policy to SCHED_IDLE
 -o, --other          set policy to SCHED_OTHER
 -r, --rr             set policy to SCHED_RR (default)
```

If the only one we support so just lets remove this `PtpSchedulingPolicy` from the API